crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
I would rather not TBH. Just seems like fluff and an extra step we would put on the consumer.
No reason to add all this. One thing we could do is mention in the docs that it will coerce  your return value into a Boolean. 
The point of this framework is not really for correctness, it is to make things easier on the dev. We say that we are going to return a Boolean, so a Boolean we must return.